lib/atlas/scaler.rb

```diff
@@ -0,0 +1,68 @@
+module Atlas
+  class Scaler
+    UNSCALED_ETENGINE_DATA_FILES = %w( fce load_profiles network ).freeze
+
+    def initialize(base_dataset_key, derived_dataset_name, number_of_residences, base_value = nil)
```

Comment:
@ploh Why is `base_value` used as the 4th optional argument? It won't be picked up by `rake scale` as far as I know.